lib/routes/auth.js

```diff
@@ -71,8 +72,36 @@ router
         return sendUser(res, user);
       })
       .catch(next);
+  })
 
+  .put('/users/:id/roles/:role', ensureAuth(), ensureRole('admin'), ({ params }, res, next) => {
```

Comment:
you don't need to pass in admin to ensureRole because your function isn't written to take a parameter